sources/engine/Stride.Graphics/Direct3D/SwapChainGraphicsPresenter.Direct3D.cs

```diff
@@ -239,7 +239,7 @@ protected internal override void OnDestroyed()
             backBuffer.OnDestroyed();
             backBuffer.LifetimeState = GraphicsResourceLifetimeState.Destroyed;
 
-            swapChain.Dispose();
+            swapChain?.Dispose();
```

Comment:
In which case `swapChain` would be null at that point ?

Reply:
Ah, looks like I should have been more curious about this one. It is another victim of being needlessly called twice for some reason causing the null exception.

Reply:
Ok, so I did some more testing today and found out it was the `ResumeManager` that was calling the second time. I removed it since the presenter gets destroyed before `ReumeManager` gets called which seems to be why it was there based on the comment. 

I don't see anymore issues but I don't know when the `ResumeManager` is used so I could be missing something here.

Reply:
Looks like the `ResumeManager` is maintained by the `GameBase` which is also where the Graphics device was being disposed so I think its good to go.